src/Ninja/G1CP/Content/Fixes/Session/fix225_MordragNcGetOut.d

```diff
@@ -0,0 +1,48 @@
+/*
+ * #225 Mordrag can be told to "get out" in New Camp
+ */
+func int G1CP_225_MordragNcGetOut() {
+    const string CONDITION = ""; CONDITION = "Org_826_Mordrag_HauAb_Condition";
+
+    if (!G1CP_IsFunc(CONDITION, "int|none"))
```

Comment:
I am all for avoiding redundancy and all, but don't you think that is a bit overkill. It obscures readability and makes the function longer and more work to type up simple fixes, while not really helping much for redundancy. Aside from that, the syntax is really weird here (why not define the constant directly), and more confusing then helping. I strongly advise to just write out the string in both occurrences, as done everywhere else in the scripts of similar fixes.